index.d.ts

```diff
@@ -0,0 +1,61 @@
+export interface CallSite {
+	/**
+	 * Returns the value of `this`.
+	 */
+	getThis(): unknown | undefined;
+	/**
```

Comment:
Should we start having an empty line between for readability?

Like:

```typescript
/**
 * Returns the value of `this`.
 */
getThis(): unknown | undefined;

/**
 * Returns the type of `this` as a string. This is the name of the function stored in the constructor field of `this`, if available, otherwise the object's `[[Class]]` internal property.
 */
getTypeName(): string | null;
```

Reply:
Yes, makes sense.